test/functional/apps/discover/_discover.js

```diff
@@ -19,20 +18,20 @@ bdd.describe('discover app', function describeIndexTests() {
     // delete .kibana index and update configDoc
     return esClient.deleteAndUpdateConfigDoc({'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*'})
     .then(function loadkibanaIndexPattern() {
-      common.debug('load kibana index with default index pattern');
+      PageObjects.common.debug('load kibana index with default index pattern');
```

Comment:
While we're at it, might as well creat an additional service for the common "page", it really doesn't feel like a `PageObject` but a collection of utilities